### DISCUSSION TITLE
[Question][Hw3] When is out & due?

### DISCUSSION THREAD

**Discussion by retroinspect** [prosyslab-classroom/cs424-program-reasoning#316] [Q&A]
By README, it seems HW3, LLVM verifer is planned to be out before midterm exam. 

Can you announce when is out and due for planning after midterm?

Thank you.

**Comment by KihongHeo** [comment]
Hi. It will be out after covering slides lecture8.pdf.
So probably, it will be out this Thursday and 2 weeks will be given.

**Comment by retroinspect** [comment]
Thanks!
